dev/Telemetry/RuntimeProfiler.cpp

```diff
@@ -73,6 +73,73 @@ namespace RuntimeProfiler {
             }
         }
         
+        //  This function returns the start of the string presentation of
+        //  given number, it assumes the buffer passed has a sufficiently
+        //  large buffer to accomodate the number.
+        static PWSTR ConstructNumber(_In_ UINT uNumber, _Out_ PWSTR pszEnd)
+        {
+            *pszEnd = 0;  // null terminate!
+
+            //  Special case 0
+            if (0 == uNumber)
+            {
+                pszEnd--;
+                *pszEnd = '0';
+            }
+            else
+            {
+                for (; uNumber; uNumber /= 10)
+                {
+                    pszEnd--;
+                    *pszEnd = '0' + (uNumber % 10);
+                }
+            }
+
+            return (pszEnd);
+        }
+
+        //  This function copies the source string to the destination string
+        //  and return the pointer to the null terminator, it never writes past
+        //  the given last character.
+        static PWSTR WriteString(_Out_ PWSTR pszDst, _In_ PCWSTR pszSrc, _In_ PWSTR pszLastChar)
+        {
+            for (; *pszSrc; pszSrc++, pszDst++)
+            {
+                *pszDst = *pszSrc;
+                if (pszDst == pszLastChar)
+                {
+                    break;
+                }
+            }
+            *pszDst = 0;
+
+            return (pszDst);
+        }
+
+        //  Thin wrapper around the firing of the event, since we may now fire
+        //  multiple events if our output buffer fills.
+        static void FireEventRaw(PCWSTR pszAPICounts, UINT32 uGroupId, UINT32 cMethods, bool bSuspend, bool bOverflow, bool bStringOverflow)
```

Comment:
There's preexisting instances in the file for sure, but I don't think we want to be perpetuating Hungarian prefixes.